xbmc/windowing/gbm/VideoSyncGbm.h

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (C) 2005-2021 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "windowing/VideoSync.h"
+#include "guilib/DispResource.h"
+
+class CVideoSyncGbm : public CVideoSync, IDispResource
+{
+public:
+  CVideoSyncGbm(void *clock) : CVideoSync(clock) {};
+  virtual bool Setup(PUPDATECLOCK func);
```

Comment:
Please drop `virtual` and add `override at end. Please do this for all the following overrides.